src/components/structures/MessagePanel.js

```diff
@@ -388,6 +388,8 @@ module.exports = React.createClass({
                 isVisibleReadMarker = visible;
             }
 
+            // XXX: there should be no need for a ghost tile - we should just use a
```

Comment:
ok, but I don't understand why that means you don't need a ghost. It exists to hold the animation.